Apps/WebClient/src/ClientApp/app/components/navmenu/navHeader.vue

```diff
@@ -1,5 +1,26 @@
+<style lang="scss" scoped>
+nav {
+  z-index: 9999;
```

Comment:
We should likely move this to a global as we reference in multiple spots.

Reply:
added variables